core/src/filed/dir_cmd.h

```diff
@@ -32,6 +34,9 @@ void* process_director_commands(JobControlRecord* jcr, BareosSocket* dir);
 void* handle_director_connection(BareosSocket* dir);
 bool StartConnectToDirectorThreads();
 bool StopConnectToDirectorThreads(bool wait = false);
+JobControlRecord* NewFiledJcr();
```

Comment:
This was a static function and is now still only referenced once in dir_cmd.cc
I think we can delete this and add static again. (see [2])